tests/test_commands.py

```diff
@@ -0,0 +1,78 @@
+import os
+import zipfile
+
+from django.core.files import File
+from django.core.management import call_command
+from django.test import TestCase
+
+from freezegun import freeze_time
+from privates.test import temp_private_root
+
+from zgw_consumers.constants import CertificateTypes
+from zgw_consumers.models import Certificate
+
+TEST_FILES = os.path.join(os.path.dirname(os.path.abspath(__file__)), "data")
```

Comment:
please use the modern `from pathlib import Path` - you can then do `Path(__file__).parent / "data"` and constructs like that :)

Reply:
refactored this for the other tests as well